website/templates/profile/account.mako

```diff
@@ -5,39 +5,101 @@
-                        </div>
-                        <button type="submit" class="btn btn-default">Update password</button>
-                    </form>
+            <div class="col-md-6">
+                <div id="connectedEmails" class="panel panel-default">
+                    <div class="panel-heading"><h3 class="panel-title">Connected Emails</h3></div>
+                    <div class="panel-body">
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th colspan="2">Primary Email</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <tr>
+                                    <td>{{ profile().primaryEmail().address }}</td>
+                                    <td></td>
+                                </tr>
+                            </tbody>
+                        </table>
+
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th colspan="3">Alternate Emails</th>
+                                </tr>
+                            </thead>
+                            <tbody data-bind="foreach: profile().alternateEmails()">
+                                <tr>
+                                    <td style="width:100%">{{ $data.address }}</td>
+                                    <td><a data-bind="click: $parent.makeEmailPrimary">make&nbsp;primary</a></td>
+                                    <td><a data-bind="click: $parent.removeEmail"><i class="fa fa-times text-danger"></i></a></td>
+                                </tr>
+                            </tbody>
+                        </table>
+
+                        <table class="table">
+                            <thead>
+                                <tr>
+                                    <th colspan="2">Unconfirmed Emails</th>
+                                </tr>
+                            </thead>
+                            <tbody>
+                                <!-- ko foreach: profile().unconfirmedEmails() -->
+                                <tr>
+                                    <td style="width:100%">{{ $data.address }}</td>
+                                    <td><a data-bind="click: $parent.removeEmail"><i class="fa fa-times text-danger"></i></a></td>
+                                </tr>
+                                <!-- /ko -->
+                                <tr>
+                                    <td colspan="2">
+                                        <form data-bind="submit: addEmail">
+                                            <div class="form-group">
+                                                <input data-bind="value: emailInput" class="form-control">
```

Comment:
You can take advantage of the HTML5's built-in validation by adding `type="email"`.

Also, might be helpful to add a placeholder, like `placeholder="New email"`.